app/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+if [ "$DATABASE" = "postgres" ]
+  then
+    echo "Waiting for psql"
+
+    while ! nc -z $SQL_HOST $SQL_PORT; do
+      sleep 0.1
+    done
+
+    echo "Psql started"
+fi
+
+python app/manage.py makemigrations
+python app/manage.py migrate
+python app/manage.py collectstatic --noinput
```

Comment:
why you need that? your djagno doesn't host any static files

Reply:
this is for admin panel